File: Beecrowd/1 - Beginner/1435 - 1985/1848 - Counting Crow/1848_CountingCrow.c
#include <stdio.h>
#include <string.h>

int main() {
    int cawCounter = 0, cawValue = 0;

    while (cawCounter < 3) {
        char caw[10];
        fgets(caw, sizeof(caw), stdin);
        caw[strcspn(caw, "\r\n")] = 0;
        
        if (strcmp(caw, "--*") == 0) cawValue += 1;
        else if (strcmp(caw, "-*-") == 0) cawValue += 2;
        else if (strcmp(caw, "-**") == 0) cawValue += 3;
        else if (strcmp(caw, "*--") == 0) cawValue += 4;
        else if (strcmp(caw, "*-*") == 0) cawValue += 5;
        else if (strcmp(caw, "**-") == 0) cawValue += 6;
        else if (strcmp(caw, "***") == 0) cawValue += 7;
        else if (strcmp(caw, "caw caw") == 0) {
            printf("%d\n", cawValue);
            
            cawValue = 0;
            cawCounter++;
        }
    }

    return 0;
}

File: Beecrowd/1 - Beginner/1435 - 1985/1848 - Counting Crow/1848_CountingCrow.py
caw_counter = 0
caw_value = 0

while caw_counter < 3:
    caw = input()

    if caw == "--*":
        caw_value += 1
    elif caw == "-*-":
        caw_value += 2
    elif caw == "-**":
        caw_value += 3
    elif caw == "*--":
        caw_value += 4
    elif caw == "*-*":
        caw_value += 5
    elif caw == "**-":
        caw_value += 6
    elif caw == "***":
        caw_value += 7
    elif caw == "caw caw":
        print(caw_value)
        
        caw_value = 0
        caw_counter += 1
